### ISSUE TITLE
ERR| vtkXOpenGLRenderWindow (0x11f80320): Could not find a decent config

### ISSUE THREAD

**Issue by gujiuren** [wzzheng/TPVFormer#37]
请问有遇到这个问题么，在运行“2. generate individual video frames”部分的指令时,碰到了这个问题。
`
visualizing scene-0916
/home/zzh/DataStack_2T/workSpace/TPVFormer/visualization/dataset.py:73: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  sweep_cams = np.array(sweep_cams)
/home/zzh/DataStack_2T/workSpace/TPVFormer/visualization/dataset.py:74: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  sweep_tss = np.array(sweep_tss)
236
processing frame 0 of scene 0
/home/zzh/anaconda3/envs/TPVFormer/lib/python3.8/site-packages/torch/utils/checkpoint.py:25: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
2955
2023-05-01 17:19:17.196 (  36.717s) [        E9797740]vtkXOpenGLRenderWindow.:266    ERR| vtkXOpenGLRenderWindow (0x11f80320): Could not find a decent config

2023-05-01 17:19:17.196 (  36.717s) [        E9797740]vtkXOpenGLRenderWindow.:484    ERR| vtkXOpenGLRenderWindow (0x11f80320): Could not find a decent visual

Aborted (core dumped)
`

**Comment by huang-yh** [comment]
We haven't run into this problem before, though it seems related with the opengl driver.
